tests/test_scrubbers.py

```diff
@@ -29,3 +30,16 @@ def test_hash_scrubber_textfield(self):
         data.refresh_from_db()
 
         self.assertNotEqual(data.description, 'Foo')
+
+    def test_faker_scrubber_date_in_past(self):
+        """
+        Use this as an example for Faker scrubbers with parameters passed along
+        """
+        data = DataFactory.create(date_past=datetime.datetime.now())
+        with self.settings(DEBUG=True, SCRUBBER_GLOBAL_SCRUBBERS={
+                'date_past': scrubbers.Faker('past_date', start_date="-30d", tzinfo=None)}):
+            call_command('scrub_data')
+        data.refresh_from_db()
+
+        self.assertGreater(datetime.date.today(), data.date_past)
+        self.assertGreater(datetime.date.today() - datetime.timedelta(days=28), data.date_past)
```

Comment:
I'm not sure this is a good idea. This test might pass by chance, if the first generated fake date just _happens_ to be further in the past than 28 days. Maybe we should use some other provider for this test? One whose behavior with and without arguments is ensured to have no overlap?

Or am I missing something?

Reply:
I mean the complementary case: the date _with parameters_ is guaranteed to be 29 days in the past, but the date _without parameters_ is *not* guaranteed to *not* be more than 29 days in the past. So the test might *pass*, even if the provider did *not* receive the parameters.

Reply:
Ahh, damn, you're right. I didn't think to test the opposite as well.
How about the ean faker.
It returns 13 digits by default and can be changed to the 8 digit version using a parameter?

Reply:
That would also not run into trouble with Postgres not casting strings to dates automatically. (which is a different issue altogether)

Reply:
Sounds good!
Let's split this: please refactor the test then we can try a second PR to add arbitrary field types.